conference.js

```diff
@@ -1938,8 +2025,37 @@ export default {
             .then(() => {
                 mediaDeviceHelper.setCurrentMediaDevices(devices);
                 APP.UI.onAvailableDevicesChanged(devices);
+                this.updateVideoIconEnabled();
             });
     },
+    /**
+     * Determines whether or not the video button should be enabled.
+     */
+    updateVideoIconEnabled() {
+        const videoMediaDevices
+            = mediaDeviceHelper.getCurrentMediaDevices().videoinput;
+        const hasPermissions
+            = videoMediaDevices !== undefined
+                && videoMediaDevices.filter(d => d.label !== '').length > 0;
+        const videoDeviceCount
+            = videoMediaDevices !== undefined ? videoMediaDevices.length : 0;
```

Comment:
Instead of checking for undefined could you just check for truthiness? I won't argue against any personal preference here, though.